momapbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Velocity Controllers ---------------------------------------
  right_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_right_front_wheel
    pid: {p: 10.0, i: 500, d: 0.00, i_clamp_min: -50, i_clamp_max: 50}
  left_front_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_left_front_wheel
    pid: {p: 10.0, i: 500, d: 0.00, i_clamp_min: -50, i_clamp_max: 50}
  right_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_right_rear_wheel
    pid: {p: 10.0, i: 500, d: 0.00, i_clamp_min: -50, i_clamp_max: 50}
  left_rear_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_to_left_rear_wheel
    pid: {p: 10.0, i: 500, d: 0.00, i_clamp_min: -50, i_clamp_max: 50}
